src/main/kotlin/com/rose/gateway/minecraft/logging/LogFormatter.kt

```diff
@@ -0,0 +1,23 @@
+package com.rose.gateway.minecraft.logging
+
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import java.util.logging.LogRecord
+import java.util.logging.SimpleFormatter
+
+/**
+ * This class determines the format of the FileLogger
+ */
+class LogFormatter : SimpleFormatter() {
+
+    override fun format(record: LogRecord): String? {
+        val zdt = ZonedDateTime.ofInstant(
+            record.instant, ZoneId.systemDefault()
```

Comment:
Either separate the arguments out to separate lines or put them on the same line as the function call they are passed to. Doing both looks weird.